src/three_by_three_board.py

```diff
@@ -1,7 +1,20 @@
+import src.constants as constants
+
 class ThreeByThreeBoard():
```

Comment:
It's meant to only be a single type of board. The type of board that is used for the game is determined by the boardBuilder, which would be called by the app class in conjunction with getting data about what type of board the user wants to play on. Currently, since there is only one type of board the cli methods default to returning the 3x3.

I might end up extracting alot of these grouped components into their own folders to help make the structure a bit more intuitive